Ginger/Ginger/Run/GingerRunner.cs

```diff
@@ -748,9 +753,15 @@ public void RunAction(Act act, bool checkIfActionAllowedToRun = true, bool stand
                 }
 
                 OnGingerRunnerEvent(GingerRunnerEventArgs.eEventType.ActionEnd, null);
+
+                // function for action log 
+                // Add flag on GingerRunner of LogLevel
+                if (mGingerRunnerLogger == null) mGingerRunnerLogger = new GingerRunnerLogger(@"c:\temp\GingerLog.txt");  // temp move from here
```

Comment:
Please log to Ginger folder \ExecutionLogs\
file name to be with date and time: yyyymmdd_hhhhmm